lncfg/db.go

```diff
@@ -215,6 +220,11 @@ type DatabaseBackends struct {
 	// the underlying wallet database from.
 	WalletDB btcwallet.LoaderOption
 
+	// NativeSQLStore is a pointer to a native SQL store that can be used
+	// for native SQL queries for tables that already support it. This may
+	// be nil if the use-native-sql flag was not set.
+	NativeSQLStore *sqldb.BaseDB
```

Comment:
I think we want to instead point to the main interface? So either the `sqlc` level one, or the new interface in `invoices` that it implements.